src/main/java/com/github/andriiyan/sprongtraining/impl/utils/file/serializer/ByteSerializer.java

```diff
@@ -23,7 +23,6 @@ public TerminalObject(long mTerminalUUID) {
             this.mTerminalUUID = mTerminalUUID;
         }
 
-        // TODO: 3/20/2022 - looks like this method is never used
         public void setmTerminalUUID(long mTerminalUUID) {
```

Comment:
Please remove unused method

Reply:
Without this method object cannot be instantiated if it read from the byte array